scripts/update_metadata.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python3
+'''
+@author: Patrick Cleeve
+
+This is a script to update the MD_POS_COR for the stage-bare component while odemis is running.
+
+run as:
+python3 ./scripts/update_metadata.py 
+
+and follow the instructions in the terminal.
+'''
+
+
+from odemis import model
+
+def main():
+
+    print(f"Calibration metadata updater")
+
+    stage = model.getComponent(role="stage-bare")
+    print(f"-"*20)
+    print(f"Current stage-bare component: {stage}")    
+    print(f"Current calibration metadata: {stage.getMetadata()[model.MD_POS_COR]}")
+
+    # get x , y values from user
+    x = input(f"Enter new x calibration value: ")
+    y = input(f"Enter new y calibration value: ")
+    print("-"*20)
+
+    # check they are floats
+    try:
+        x = float(x)
+        y = float(y)
```

Comment:
Wasn't there a plan to divide by 2?

Reply:
I think we should change this calibration value to just be the distance between the FIB and METEOR, rather than half it. The reason it was half was to match the other rotation calibration, which used the radius and then multipled by two, but there is no real reason for them to be the same? 

I think it being half makes it really confusing for the user either way because they either have to measure the distance then divide by two (for some unknnown reason to them), or they enter the actual distance (e.g. 4) and then get a message saying there calibration value has been set to 2. 


I think the real solution will be to add an additional calibration parameter to separate the calibration for rotated movements, and non-rotated moves, or somehow unify them. That would avoid this issue entirely, and make a better user experience because they could move from SEM <>METEOR or FIB <> METEOR without having to redo calib?
Im happy to implement that change in this PR, or merge this and add it separately?

Reply:
@pieleric  I've changed the calibration for the no rotation move to not multiple by two (so dont need to divide the distance by two). I think this will be much less confusing for users. 

I've added a check in the calibration script to check that the values are within the stage axes (should prevent wildly bad values due to units etc), and to check the transform values are within the ranges defined for SEM <> FM imaging. It also checks that the yaml is setup for no rotation movements, and won't let you change the values if not. I've added a warning on the actual transformation calc if the transformation puts the stage outside the imaging range, which can be changed to raise an error if we want to be stricter there.  

I think this should prevent the worst errors for now as its only for 1 system, and we can re open this as part of the new movement system.